plutus-tx/src/PlutusTx/Builtins.hs

```diff
@@ -82,6 +82,7 @@ module PlutusTx.Builtins (
                          , headMaybe
                          , BI.head
                          , BI.tail
+                         , BI.drop
```

Comment:
Should this also go in [PlutusTx.Prelude](https://github.com/IntersectMBO/plutus/blob/master/plutus-tx/src/PlutusTx/Prelude.hs)? I seem to remember that the contents of that file are kind of random though.